app/main.js

```diff
@@ -161,6 +165,32 @@ function startProcessWin () {
   })
 }
 
+
+function createWindow() {
```

Comment:
we should use some more meaningful name, eg `showWelcomeWindow`, so next contributors know what is happening here.